avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -337,17 +336,17 @@ mucoulomb = 0.155
 mu0wetsnow = 0.2
 # xsiWetSnow
 xsiwetsnow = 981.
-# reference enthalpy in J/kg to compute the new mu = mu_0 x exp(-enthalpy/enthRef)
+# reference enthalpy to compute the new mu = mu_0 x exp(-enthalpy/enthRef) [J/kg]
 enthRef = 20000
-# initial temperature in °C
+# initial temperature [°C]
 TIni = -10
-# initial temperature for entrainment in °C
+# initial temperature for entrainment [°C]
 entTempRef = -10.0
 # J/kgK (ice)
 cpIce = 2050.0
```

Comment:
I believe this parameter is not explained in the documentation https://docs.avaframe.org/en/1.8.3/theoryCom1DFA.html#wet-snow-friction-type has this been added recently?